Makefile

```diff
@@ -4,6 +4,7 @@ PACKAGES := $$(go list ./...| grep -vE 'vendor')
 
 GOFILTER := grep -vE 'vendor|render.Delims|bindata_assetfs|testutil'
 GOCHECKER := $(GOFILTER) | awk '{ print } END { if (NR > 0) { exit 1 } }'
+GOVERSION := $(shell go version | awk '{ printf "%5.5s", $$3 }')
```

Comment:
`GOVERSION` should be `go1.7`, but I got `go ve%`.

OS: ubuntu 14 LTS
awk:  1.2

```
neil:~/ $ go version | awk '{ printf "%5.5s", $$3 }'
go ve%   
```